script/DeployArbitrumPayload.sol

```diff
@@ -0,0 +1,25 @@
+// SPDX-License-Identifier: MIT
+pragma solidity ^0.8.0;
+import {Script} from 'forge-std/Script.sol';
+
+// import {AaveV3ArbCapsPayload} from '../src/contracts/arbitrum/AaveV3ArbCapsPayload.sol';
+
+contract DeployArbitrumPayload {
```

Comment:
As this behavior is quite general, should you just inherit from a base contract `WithChainIdValidation`?